nservicebus/pipeline/satellites.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating Satellites
+summary: Extension point for handling messages
+tags: 
+related:
+ - nservicebus/pipeline/customizing-v6
+ - nservicebus/pipeline/features
+redirects:
+---
+
+In NServiceBus Versions 5 and below, Satellites were extension points to handle messages that arrive in different queues other than the endpoint's main queue. This extension point came in handy when the robustness of a separate queue was needed without having to create and setup a brand new Endpoint and configure any needed message mappings.
```

Comment:
@mikeminutillo @SimonCropp - 

>Should we use past tense here? This is still the way to do things in v5. Also, are Satellites still an abstraction? This opening paragraph makes it sounds like they are gone but then the rest of the document explains how to set one up.

The satellites were never documented. It is now gone as of V6. The idea was we were going to do this in the upgrade guide. As @petersgiles said, this is a migration guide for users that are currently using the satellites to move over to the pipeline.

The interfaces are gone. But for users that were using them in v6, this is a step by step guide on how to move it over to the new feature / pipeline setup.